Framework/API/test/PreviewManagerTest.h

```diff
@@ -57,15 +58,29 @@ class PreviewManagerTest : public CxxTest::TestSuite {
     auto previews = PreviewManager::Instance().getPreviews("TestFacility", "Crystal");
     TS_ASSERT(previews.empty());
   }
+  void test_get_preview_by_facility_and_technique_and_acquisition() {
```

Comment:
These tests can be reformatted to not include the acquisition, perhaps the on on `non_existent_acquisition` can be removed altogether.

Reply:
I believe all the tests including acquisition can be straight deleted, since removing the acquisition would make them redundant.